react/I18n/format.spec.jsx

```diff
@@ -75,11 +78,189 @@ describe('formatLocallyDistanceToNowStrict', () => {
     expect(result).toEqual('1 hour')
   })
 
+  it('should formatDistanceToNowStrict with one day', () => {
+    const date = Date.now() + 24 * 60 * 60 * 1000 // 1d
+
+    const result = formatLocallyDistanceToNowStrict(date)
+
+    expect(result).toEqual('1 day')
+  })
+
+  it('should formatDistanceToNowStrict with two days', () => {
+    const date = Date.now() + 2 * 24 * 60 * 60 * 1000 // 2d
+
+    const result = formatLocallyDistanceToNowStrict(date)
+
+    expect(result).toEqual('2 days')
+  })
+
   it('should formatDistanceToNowStrict with very high value', () => {
     const date = Date.now() + 42 * 24 * 60 * 60 * 1000 // 42d
 
     const result = formatLocallyDistanceToNowStrict(date)
 
     expect(result).toEqual('1 month')
   })
+
+  describe('for hours', () => {
+    afterEach(() => {
+      MockDate.reset()
+    })
+
+    it('should return 1 hour', () => {
+      MockDate.set('2023-01-01T00:00:00')
+      const date = new Date('2023-01-01T01:00:00')
+
+      const result = formatLocallyDistanceToNowStrict(date)
+
+      expect(result).toEqual('1 hour')
+    })
+
+    it('should return 23 hours', () => {
+      MockDate.set('2023-01-01T00:00:00')
+      const date = new Date('2023-01-01T23:59:00')
+
+      const result = formatLocallyDistanceToNowStrict(date)
+
+      expect(result).toEqual('23 hours')
+    })
+
+    it('should return 11 hours', () => {
+      MockDate.set('2023-01-01T12:00:00')
+      const date = new Date('2023-01-01T23:59:00')
+
+      const result = formatLocallyDistanceToNowStrict(date)
+
+      expect(result).toEqual('11 hours')
+    })
+  })
+
+  describe('for days', () => {
+    describe('at midnight', () => {
+      beforeEach(() => {
+        MockDate.set('2023-01-01T00:00:00')
+      })
+
+      afterEach(() => {
+        MockDate.reset()
+      })
+
+      it('should return 1 day', () => {
+        const date = new Date('2023-01-02T00:00:00')
+
+        const result = formatLocallyDistanceToNowStrict(date)
+
+        expect(result).toEqual('1 day')
+      })
+
+      it('should return 1 day', () => {
+        const date = new Date('2023-01-02T01:00:00')
+
+        const result = formatLocallyDistanceToNowStrict(date)
+
+        expect(result).toEqual('1 day')
+      })
+
+      it('should return 1 day', () => {
```

Comment:
nit: Different names for these tests to be able to find them faster if they fail?

Reply:
j'ai renommé en ajoutant l'heure utilisée dans le test